packages/core/integration-tests/test/sourcemaps.js

```diff
@@ -2,6 +2,7 @@ const assert = require('assert');
 const fs = require('@parcel/fs');
 const path = require('path');
 const mapValidator = require('sourcemap-validator');
+const SourceMap = require('../src/SourceMap');
```

Comment:
Looks like the tests are failing because of this import. Maybe this test should be inside the `parcel-bundler` package rather than the integration tests?

Reply:
It probably should although maybe it would be a good idea to extract the sourcemap implementation in the future to allow plugins to use it?

Reply:
yeah eventually